src/gui/editors/AutomationEditor.cpp

```diff
@@ -66,6 +66,7 @@
 
 QPixmap * AutomationEditor::s_toolDraw = NULL;
 QPixmap * AutomationEditor::s_toolErase = NULL;
+QPixmap * AutomationEditor::s_toolDrawOut = NULL;
```

Comment:
We should use `nullptr` instead of `NULL`.

Reply:
Changed all places `NULL` was used for pointers to `nullptr` in that file.